scripts/singlecell/geneformer/train.py

```diff
@@ -138,14 +149,28 @@ def main(
         tensor_model_parallel_size=tensor_model_parallel_size,
         pipeline_model_parallel_size=pipeline_model_parallel_size,
     )
+    if aligned_megatron_ddp:
+        ddp: str | DistributedDataParallelConfig = DistributedDataParallelConfig(
```

Comment:
I tried to comment it in the else block. What "megatron" does behind the scenes is get changed into:
```# this will launch DistributedDataParallelConfig(check_for_nan_in_grad=True).```

So same thing but with different settings. I pulled this version with these settings from many of the nemo2 recipes. It doesn't seem to make a big difference, but maybe it will if we get past whatever other cluster issues are slowing us down.